containers/daphne-dev.Dockerfile

```diff
@@ -34,7 +34,16 @@ RUN apt-get -qq -y update && apt-get -y upgrade && apt-get -y --no-install-recom
     vim nano rsync sudo iputils-ping virtualenv openssh-server iproute2 git htop gdb lldb lld gpg-agent net-tools \
     software-properties-common ca-certificates file unzip wget tar zstd \
     ccache python3-pip python3-networkx python3-dev graphviz-dev \
+    lsb-release gnupg \
     && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+RUN set -xe && wget https://apt.llvm.org/llvm.sh && \
+    chmod +x llvm.sh && \
+    ./llvm.sh 18
+
+RUN set -xe &&  \
+    apt-get -y --no-install-recommends install clang-format-18
+
```

Comment:
Maybe we should add this method of installation of llvm/clang to the documentation section ?

Reply:
See above, I'll remove this.